modules/heat_conduction/doc/content/source/kernels/HomogenizedHeatConduction.md

```diff
@@ -1,14 +1,27 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # HomogenizedHeatConduction
 
-!alert construction title=Undocumented Class
-The HomogenizedHeatConduction has not been documented. The content contained on this page
-includes the basic documentation associated with a MooseObject; however, what is contained is
-ultimately determined by what is necessary to make the documentation clear for users.
-
 !syntax description /Kernels/HomogenizedHeatConduction
 
+## Description
+
+This `Kernel` computes
+\begin{equation}
+\int_Y \frac{\partial v}{\partial y_i}k_{ik} \textrm{d}y
+\end{equation}
+where $k$ is the diffusion coefficient (thermal conductivity).  It is used in conjunction with the [Heat Conduction](HeatConduction.md) `Kernel` and the [Homogenized Thermal Conductivity](HomogenizedThermalConductivity.md) `PostProcessor` to compute homogenized thermal conductivity values according to
+\begin{equation}
+\int_Y \frac{\partial v}{\partial y_i}k_{ik} \frac{\partial \psi^k}{\partial y_j} = \int_Y \frac{\partial v}{\partial y_i}k_{ik} \text{d}y
+\end{equation}
+and
+\begin{equation}
+k_{ij}^\text{H}=\frac{1}{\left|\text{Y}\right|}\int_\text{Y}k_{ij}\left(\bm{I} + \frac{\partial\psi^k}{\partial y_j}\right)\;\text{d}\bm{y}
+\end{equation}
+where $k^H_{ij}$ is the homogenized thermal conductivity.  See [cite](hales15homogenization).
+
+## Example Input File Syntax
+
+!listing modules/combined/test/tests/homogenization/heatConduction2D.i block=Kernels/HomogenizedHeatConduction
```

Comment:
Changing `block=Kernels/HomogenizedHeatConduction` to either `block=Kernels/heat_rhs_x` or `block=Kernels/heat_rhs_y` should fix the failure.

Reply:
Thanks.